shaders/src/main.fs

```diff
@@ -28,6 +28,17 @@ void main() {
 
     fragColor = evaluateMaterial(inputs);
 
+#if defined(HAS_FOG)
+    // we need the real world position, which is obtained by adding the world-offset
+    vec3 wsCameraPostion = getWorldCameraPosition() + getWorldOffset();
```

Comment:
```suggestion
    vec3 worldCameraPostion = getWorldCameraPosition() + getWorldOffset();
```

Reply:
I've used "ws", "cs", "vs" prefixes elsewhere to mean "world-space", "clip-space", "view-space" respectively.  This is the "camera position expressed in world-space coords".

Reply:
I understand what it means, but we don't have those abbreviations really in the shaders.

Reply:
well, I started using that convention for new code. it was very useful w/ contact shadows where 4 different spaces were used. I think it's a good convention going forward.

Reply:
I don't like abbreviations. And I'm all for naming things based on which space they're in btw (hence my proposal of the "world" prefix).